src/pages/workspace/members/WorkspaceMemberDetailsPage.tsx

```diff
@@ -89,11 +90,22 @@ function WorkspaceMemberDetailsPage({personalDetails, policy, route}: WorkspaceM
     }, [policyID, workspaceAccountID]);
 
     const memberCards = useMemo(() => {
-        if (!allCardsList) {
+        if (!cards && !expensifyCards) {
             return [];
         }
-        return Object.values(allCardsList ?? {}).filter((card) => card.accountID === accountID && workspaceAccountID.toString() === card.fundID);
-    }, [allCardsList, accountID, workspaceAccountID]);
+        // For admin Expensify Cards can also appear in the cards list, so we need to remove duplicates
+        const allCards = [...Object.values(cards ?? {}), ...Object.values(expensifyCards ?? {})];
+        const cardIDs = new Set();
+        const uniqueObjects = allCards.filter((obj) => {
+            if (cardIDs.has(obj.cardID)) {
+                return false;
+            }
+            cardIDs.add(obj.cardID);
+            return true;
+        });
+
+        return Object.values(uniqueObjects ?? {}).filter((card) => card.accountID === accountID && workspaceAccountID.toString() === card.fundID);
```

Comment:
Card doesn't display on the member detail page, because `fundID` isn't exist on expensifyCards

https://github.com/user-attachments/assets/45516cdf-dee3-40a9-b0e9-833c368f0c28


@koko57 Do you know why we `add workspaceAccountID.toString() === card.fundID` condition before?

cc

Reply:
not sure, I just left this condition as it was

Reply:
ECards should have `fundID` too 🤔

Reply:
ref: https://github.com/Expensify/App/pull/49928#discussion_r1823773802